common/djangoapps/enrollment/urls.py

```diff
@@ -27,6 +28,9 @@
         name='courseenrollment'
     ),
     url(r'^enrollment$', EnrollmentListView.as_view(), name='courseenrollments'),
+
+    url(r'^enrollment/bulk$', BulkEnrollmentView.as_view(), name='bulk_course_enrollments'),
```

Comment:
The url doesnt make much sense, I believe you should update it to either `enrollments$` or to `enrollment/create_many_enrollments`